packages/blockchain-wallet-v4-frontend/src/scenes/Home/PriceChart/Chart/selectors.ts

```diff
@@ -7,23 +7,21 @@ import { selectors } from 'data'
 export const getData = createDeepEqualSelector(
   [
     selectors.core.settings.getCurrency,
-    selectors.preferences.getPriceChart,
+    selectors.components.priceChart.getTime,
```

Comment:
`preferences.getPriceChart` was updated, via the `preferences` reducer, with the same actions used in the `priceChart` reducer - so in this case the data obtained was exactly the same